src/funcx_common/messagepack/message_types/task_transition.py

```diff
@@ -0,0 +1,9 @@
+from ...tasks.constants import ActorName, TaskState
+from .base import Message, meta
+
+
+@meta(message_type="task_transition")
+class TaskTransition(Message):
+    timestamp: int
```

Comment:
Nit: I assume that this is intended to be epoch time, but it isn't obvious/explicit